packages/@lightningjs/ui-components/src/components/MetadataBase/MetadataBase.js

```diff
@@ -186,6 +189,38 @@ class MetadataBase extends Base {
     });
   }
 
+  _updateSubtitle() {
+    if (!this.subtitle && !this._Subtitle) {
+      return;
+    }
+
+    if (!this._Subtitle) {
+      this._Text.childList.addAt(
+        {
+          ref: 'Subtitle',
+          type: TextBox,
+          signals: {
+            textBoxChanged: '_subtitleLoaded'
+          }
+        },
+        1
+      );
+    }
+
+    this._Subtitle.patch({
+      content: this.subtitle,
+      marquee: this.marquee,
+      style: {
+        textStyle: {
```

Comment:
my only thoughts is that the subtitle overlaps with the icons in the details. Should we make the lineheight a little bit larger so this doesn't overlaP?

Reply:
I think we might want to leave this as is. The InlineContent uses the line height of the text, so I think the Icon being taller is a side effect of how InlineContent is set up and not necessarily of how it is being used here.